### ISSUE TITLE
`keyword-spacing`: error not reported for catch, with after: true.

### ISSUE THREAD

**Issue by esthezia** [eslint-stylistic/eslint-stylistic#987]
### Validations

- [x] Follow our [Code of Conduct](https://github.com/eslint-stylistic/eslint-stylistic/blob/main/CODE_OF_CONDUCT.md)
- [x] Read the [Contributing Guide](https://eslint.style/contribute/guide).
- [x] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

### Describe the bug

Hi!

With only this trule configured for eslint (so there aren't any conflicts with other rules):

```
"stylistic/keyword-spacing": ["error", {
    "before": true,
    "after": true
 }]
```

the code block:

```
try {
    switch (aaa) {
        case 1:
            console.log("aaa");
            break;
        case 2:
            console.log("bbb");
            break;
    }
} catch(e) {}
```

doesn't report an error, where an error was expected due to the missing space after the "catch" keyword.

Thank you!

### Reproduction

<none>

### Contributes

- [ ] I am willing to submit a PR to fix this issue
- [ ] I am willing to submit a PR with failing tests

**Comment by 9romise** [comment]
This case is handled in [`space-before-function-paren`](https://eslint.style/rules/space-before-function-paren)

See: https://eslint.style/rules/keyword-spacing#keyword-spacing
